aggregator/src/aggregator.rs

```diff
@@ -3586,7 +3576,11 @@ impl VdafOps {
             {
                 let aggregate_share_expiry_time =
                     batch_interval.end().add_duration(report_expiry_age)?;
-                if clock.now().is_after(&aggregate_share_expiry_time) {
+                if clock
+                    .now()
+                    .to_time(task.time_precision())
+                    .is_after(&aggregate_share_expiry_time)
```

Comment:
I wonder if we could make this (and several other places where we compare times) more ergonomic by defining `is_after`/`is_before` methods on `TimeExt` that deal with `DateTime<Utc>`.

Reply:
I did this a little differently than that, but see in 705d0622.